### DISCUSSION TITLE
How to create a model for fine-tuning through multiple passthrough merges?I want to create 5b by merging qwen2.5 1.5b, can anyone guide me?

### DISCUSSION THREAD

**Discussion by win10ogod** [arcee-ai/mergekit#436] [General]
How to create a model for fine-tuning through multiple passthrough merges?
I want to create 5b by merging qwen2.5 1.5b, can anyone guide me?

**Comment by jim-plus** [comment]
The paper for SOLAR 10.7B indirectly provides a formula for upscaling from 7B (or 7.2B) to 10.7B (or 11B).
https://arxiv.org/abs/2312.15166
